content/blog/2015/2015-12-03-pipeline-as-code-with-multibranch-workflows-in-jenkins.md

```diff
@@ -84,6 +84,11 @@ Also, ensure that other dependencies, like SCM plugins and build tools, are met:
 * GitHub Branch Source Plugin (optimized to use the GitHub API and improve performance)
 * Maven build tool
 
+Finally, make sure you've created the required Webhook from your SCM (Github in this case) to Jenkins.
+Here's how to do that: 
+https://thepracticalsysadmin.com/setting-up-a-github-webhook-in-jenkins/
```

Comment:
Oops, copy-paste! missed out on embedding those as links. Let me make that change and generate a new PR.